script.js

```diff
@@ -39,17 +49,20 @@ document.addEventListener("DOMContentLoaded", function () {
 		task.value = '';
 	});
 
-	arrayTasks2.forEach(function (e) {
+	arrayTasks2.forEach((e) => {
 		liMaker(e)
 	});
 
 	/*
 	 * Usuwamy ostatni element
 	 */
-	btnDel.addEventListener('click', function () {
-		let lastTask = document.querySelector('ul li:last-child')
+	btnDel.addEventListener('click', () => {
+		let lastTask = document.querySelector('ul li:last-child');
+		lastTask.classList.add('removeLi');

```

Comment:
@PrzemekBartecki wouldn't be better to call it in `if` block to make sure this variable is available